testing.md

```diff
@@ -61,5 +171,55 @@ The following query is sufficient to test Native Query as it exercises mutliple
 | laptop | 3257.093 |
 | envelopes | 77.661 |
 
+# Manual Testing (for Direct Query)
+
+At this time, Direct Query must be tested semi-manually by reviewers, until such time as the Power Query
+SDK uses the same Mashup Engine as Power BI.
 
+Each Direct Query test we want to run is stored in the `resources/direct_query` directory
+
+1. Ensure you have a local ADF running using the instructions from the [Setup](Setup) Section.
+- Load data
+    ```
+    ./mongoimport.exe --uri="mongodb://localhost:28017/reports" --drop resources/integration_test/testdata/transforms.json
+    ./mongoimport.exe --uri="mongodb://localhost:28017/reports" --drop resources/integration_test/testdata/table_ops.json
+    ```
+  - Set the ADF schema
```

Comment:
When I look at the rendered md file, this is a sub-bullet. Not sure it's what you intended.

Reply:
I intended it to be a sub bullet under "load the data", but I'm not stuck on this idea.